mono/utils/mach-support-arm.c

```diff
@@ -25,24 +25,6 @@
        #define __darwin_mcontext       __darwin_mcontext32
 #endif
 
-/* Known offsets used for TLS storage*/
-
-
-static const int known_tls_offsets[] = {
-	0x48, /*Found on iOS 6 */
-	0xA4,
-	0xA8,
-};
-
-#define TLS_PROBE_COUNT (sizeof (known_tls_offsets) / sizeof (int))
-
-/* This is 2 slots less than the known low */
-#define TLS_PROBE_LOW_WATERMARK 0x40
-/* This is 24 slots above the know high, which is the same diff as the knowns high-low*/
-#define TLS_PROBE_HIGH_WATERMARK 0x108
-
-static int tls_vector_offset;
-
 void *
 mono_mach_arch_get_ip (thread_state_t state)
```

Comment:
`mono_mach_arch_get_ip` and `mono_mach_arch_get_sp` seems dead too.

Reply:
I can remove them too if desired. Originally I just wanted to remove the TLS code and keep the patch small because it was dead code that was actively causing trouble. These functions are harmless compared to that.

Reply:
If the code is dead, then it's ok to remove it. It will always be part of the history so it's not completely lost anyway.